### PR TITLE
Add icon for Alaska

### PR DESCRIPTION
**Pull request by service-paradis**
# Description

Add icon for Alaska (https://play.google.com/store/apps/details?id=com.editionsalaska.ra)

### Added
Alaska (`com.editionsalaska.ra`)

## Contributor's checklist

- [x] I have followed the [Lawnicons Guidelines](https://github.com/LawnchairLauncher/lawnicons/blob/develop/CONTRIBUTING.md)
- [x] I have ensured that Lawnicons builds correctly
- [x] I am willing to make changes to my icons if someone suggests changes

### CODE REVIEW

File: svgs/alaska.svg
Comment (x9136) [inline svgs/alaska.svg:1]: It's worth it to reduce the black spot somehow.

Screenshot 2024-01-20 at 17 02 45

We have icons with this same case: [bob_world.svg](https://github.com/LawnchairLauncher/lawnicons/blob/0911e4b183fb633f52473a66be38cc15976b776f/svgs/bob_world.svg), [zlib.svg](https://github.com/LawnchairLauncher/lawnicons/blob/0911e4b183fb633f52473a66be38cc15976b776f/svgs/zlib.svg).

**Review by x9136 — CHANGES_REQUESTED** [review]

File: svgs/alaska.svg
Comment (service-paradis) [reply svgs/alaska.svg:1]: Makes sense, thank you for the feedback. Just pushed a new version using `6px` stroke for the inner pages!

**Review by x9136 — APPROVED** [review]
Added 1 pixel to the center line because you didn't increase the stroke length after changing the stroke thickness.

**Comment by service-paradis** [comment]
Oh, damn 🤦‍♂️ Thank you for the quick fix!

**Pull request merged**